### PR TITLE
feat: display organization names in previously logged in list

### PR DESCRIPTION
**Pull request by kimskimchi**
Display organization names on the list of previously visited organization login pages. 

image

### CODE REVIEW

File: ui/pages/login/organizations.js [hunk 1]
```diff
@@ -28,6 +28,8 @@ export default function Organizations() {
             key={o.url}
             className='my-2 inline-flex w-full items-center justify-between rounded-md border border-gray-300 bg-white py-2.5 px-4 text-xs text-gray-500 shadow-sm hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'
           >
+            {o.name}
```
Comment (hoyyeva) [inline ui/pages/login/organizations.js:31]: lets style this a little bit
maybe we can convert this whole `<a />` to an `<button />`
something like this - you might want to play around a bit though 
`<button
                onClick={...}
                key={o.url}
                title={`${o.name} — ${o.url}`}
                className='my-2 inline-flex w-full items-center rounded-md border border-gray-300 bg-white py-2.5 px-4 shadow-sm hover:bg-gray-100 focus:outline-none focus:ring-2 focus:ring-blue-500 focus:ring-offset-2'
              >
                <div className='text-xs text-gray-500'>{o.name}</div>
                <div className='text-2xs text-gray-400'>{o.url}</div>
              </button>`

Comment (jmorganca) [reply ui/pages/login/organizations.js:31]: Excited to see what this looks like

File: ui/pages/login/callback.js [hunk 2]
```diff
@@ -36,8 +41,7 @@ export default function Callback() {
         router.replace('/')
       }
       window.localStorage.removeItem('next')
-
-      saveToVisitedOrgs(window.location.host, baseDomain)
+      saveToVisitedOrgs(window.location.host, baseDomain, data.organizationName)
```
Comment (BruceMacD) [inline ui/pages/login/callback.js:44]:
```suggestion
      saveToVisitedOrgs(window.location.host, baseDomain, data?.organizationName)
```

File: ui/pages/login/index.js [hunk 3]
```diff
@@ -129,7 +131,7 @@ export default function Login() {
 
       await mutate('/api/users/self')
       router.replace('/')
-      saveToVisitedOrgs(window.location.host, baseDomain)
+      saveToVisitedOrgs(window.location.host, baseDomain, data.organizationName)
```
Comment (BruceMacD) [inline ui/pages/login/index.js:134]:
```suggestion
      saveToVisitedOrgs(window.location.host, baseDomain, data?.organizationName)
```

**Review by BruceMacD — APPROVED** [review]
Back-end change looks good

**Review by hoyyeva — APPROVED** [review]

**Pull request merged**